### ISSUE TITLE
Getting an error while loading program

### ISSUE THREAD

**Issue by jedijos** [shinyquagsire23/OpenJKDF2#125]
Program doesn't launch at all, I'm getting an error while resources are being loaded:
Parse shader 'resource/shaders/default_f.glsl'
ERROR: (0)  : error c7011: implicit cast from "int" to "ivec2"

After that, it closes.
I've tried with a clean installation, with first and last releases, same error happens.
I'm on windows 10,  original jk cd release.

**Comment by shimstock22** [comment]
I’m also running Win 10 with original assets. I didn’t have the same issue, but I assume this is the GLSL shader issue he was aware of in the instructions. 
“ When running on SDL2-based builds (Linux/MacOS/Win64), be sure to have copied the GLSL shaders from resource/shaders/* to <JK.EXE base folder>/resource/shaders/*, as shown above.”
Are your shaders in the right directory?

**Comment by jedijos** [comment]
Yes, I put all files right in that path, but in that case I receive another error while loading the very first file default_v.glsl.

**Comment by jedijos** [comment]
Yes I put all files I that path, but in this case I receive another error while loading the very first file default_v.glsl.

**Comment by shimstock22** [comment]
For me, it looks like my shaders folder is in “jkgm” since I had gfxmod installed. It works for me, so I never bothered to change it. Have you tried this?

**Comment by jedijos** [comment]
I can try, could you please share the exact path?

**Comment by shimstock22** [comment]
Download this if you don’t have gfxmod yet: https://www.moddb.com/mods/jedi-knight-remastered/downloads/jedi-knight-remastered-20

It sets everything up for you. It should also create a folder called jkgm inside the main folder that has your shader folder and its contents that OpenJKDF2 might be looking for.

**Comment by jedijos** [comment]
Hello, I've tried installing the gfxmod but still have the same issue....
